### PR TITLE
Bump lcms2 to 2.9 and import test transforms from solaris-userland

### PR DESCRIPTION
**Pull request by alarcher**
ABI compatible, maintenance release.
One test failure in 32-bit, same as 2.8.0.

### CODE REVIEW

File: components/library/lcms2/Makefile [hunk 1]
```diff
@@ -46,12 +46,38 @@ CFLAGS             +=           $(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
 CXXFLAGS           +=           $(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
 LDFLAGS            +=           $(JPEG_LDFLAGS)
 
+# Seems that we have no other way to designate that we want large file support.
+CFLAGS += $(CPP_LARGEFILES)
```
Comment (pyhalov) [inline components/library/lcms2/Makefile:50]: Have you checked that this didn't break ABI?

Comment (alarcher) [reply components/library/lcms2/Makefile:50]: I only tested eom, eog and xsane with the new lcms2. What should be done?

Comment (pyhalov) [reply components/library/lcms2/Makefile:50]: Check, first of all, if previous 32-bit version of lcms2 was compiled with _FILE_OFFSET_BITS=64  (for example, in config.h).
This affects size of file-related structures (and their members) and can lead to interesting crashes. If it is not so, the safest thing to do is to build it with _FILE_OFFSET_BITS 64 and rebuild dependent apps.

Comment (pyhalov) [reply components/library/lcms2/Makefile:50]: Given that poppler depends on lcms2 and IS compiled with _FILE_OFFSET_BITS=64, I'd added this option.

**Review by pyhalov — APPROVED** [review]

File: components/library/lcms2/Makefile (hunk 1 above)
Comment (alarcher) [reply components/library/lcms2/Makefile:50]: I compared the headers and cannot find any trace of change after this flag is added, should be safe.

**Pull request merged**